README.md

```diff
@@ -55,12 +56,12 @@ Login to Industrial Edge Device and click on Icon of the Hello-World Application
 
 ## Documentation
  
-- Here is a link to the [docs](docs/) of this application example.
 - You can find further documentation and help in the following links
   - [Industrial Edge Hub](https://iehub.eu1.edge.siemens.cloud/#/documentation)
   - [Industrial Edge Forum](https://www.siemens.com/industrial-edge-forum)
```

Comment:
Could you please use the link to the new Mendix based Forum here: 
https://forum.mendix.com/link/space/industrial-edge